stdlib/public/runtime/CMakeLists.txt

```diff
@@ -62,13 +62,57 @@ set(LLVM_OPTIONAL_SOURCES
     MutexPThread.cpp
     MutexWin32.cpp
     CygwinPort.cpp
+    ImageInspectionELF.cpp
```

Comment:
When compiling for Linux it is removed from swift_runtime_sources so that it is not added to libswiftCore.a but instead used in libswiftImageInspectionShared.a 
For other ELF targets it remains in swift_runtime_source - this may change as this static option is tested on other ELF targets